lib/android/app/src/main/java/com/wix/reactnativenotifications/core/notification/PushNotification.java

```diff
@@ -207,9 +209,14 @@ private void notifyReceivedBackgroundToJS() {
 
     private void notifyOpenedToJS() {
         Bundle response = new Bundle();
-        response.putBundle("notification", mNotificationProps.asBundle());
-
-        mJsIOHelper.sendEventToJS(NOTIFICATION_OPENED_EVENT_NAME, response, mAppLifecycleFacade.getRunningReactContext());
+        try 
+        {
+           response.putBundle("notification", mNotificationProps.asBundle());
+            mJsIOHelper.sendEventToJS(NOTIFICATION_OPENED_EVENT_NAME, response, mAppLifecycleFacade.getRunningReactContext());
+        } catch (NullPointerException e) 
+        {
+            Log.e(LOGTAG, "notifyOpenedToJS: Null pointer exception");
+        }    
```

Comment:
It actually won't fix the issue but will only prevent the app from crashing: instead of receiving a callback on JS side it will only log the "null pointer exception"

Reply:
In this case @artyorsh  I would argue this solution is sufficient, taking into account:
- It has been fixed similarly in another place in the codebase, where asBundle() call can also throw null pointer exceptions/
 https://github.com/wix/react-native-notifications/blob/47161c9d9c208d3c446e64a8458be190182f166e/lib/android/app/src/main/java/com/wix/reactnativenotifications/RNNotificationsModule.java#L81-L96
- The "intended" behavior of the library is still very much intact, push notifications are received and open properly. Just on some specific platforms (android 12) in some cases, it seems that the bundle can be `null`, and it would be better to just "bail" when there is no notification to handle rather than crash the app altogether 😅 

A "root" cause for that might be more related to #837 , but @DanielEliraz  should know more about that?

Reply:
Definitely better to have it caught than having the app crashing - true. But will it mean that in the _bailed_ cases there will be no callback on JS? I think another scenario which might be related to the issue is that this function is called twice for some reasons - first time with "good" arguments and the second time with "bad". The fix will have more sense then 🙂

Reply:
after some testing, @artyorsh 's guess is right. the function is called twice, first with the null bundle, then with the correct data.

Reply:
@DanielEliraz is this being reviewed?